staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd_patch.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
I've been doing kcp for our patch files

Reply:
ack. Was fearing some verify script will get unhappy.